playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
FWIW, the original comment has been addressed and it is now in two separate playbooks, following the same pattern as others.